src/components/table/table.tsx

```diff
@@ -22,6 +21,8 @@ import { SelectableForTable } from "zapatos/schema";
 import emptyState from "../../public/img/empty-state.png";
 import Container from "@mui/material/Container";
 import Image from "next/image";
+import { SearchBar } from "../design_system/searchBar/SearchBar";
```

Comment:
curious the relative path imports here? i would tend more toward @components/design_system/searchBar/SearchBar, which then makes table.tsx moveable.

Reply:
I can definitely change that...I found that usage in another part of the app...but the @ makes more sense